SPLASH/src/game/states/GameState.cpp

```diff
@@ -872,25 +892,25 @@ void GameState::initAnimations() {
 		animationEntity22->getComponent<AnimationComponent>()->currentAnimation = animationEntity22->getComponent<AnimationComponent>()->getAnimationStack()->getAnimation(0);
 
-		auto animationEntity3 = ECS::Instance()->createEntity("animatedModel3");
-		animationEntity3->addComponent<TransformComponent>();
-		animationEntity3->getComponent<TransformComponent>()->translate(3, 4, 2);
-		animationEntity3->getComponent<TransformComponent>()->translate(110.f, 100.f, 100.f);
-		animationEntity3->getComponent<TransformComponent>()->scale(0.005f);
-		animationEntity3->addComponent<ModelComponent>(&m_app->getResourceManager().getModel("AnimationTest/BaseMesh_Anim.fbx", shader));
-		animationEntity3->getComponent<ModelComponent>()->getModel()->setIsAnimated(true);
-		animationEntity3->addComponent<AnimationComponent>(&m_app->getResourceManager().getAnimationStack("AnimationTest/BaseMesh_Anim.fbx"));
-		animationEntity3->getComponent<AnimationComponent>()->currentAnimation = animationEntity3->getComponent<AnimationComponent>()->getAnimationStack()->getAnimation(0);*/
+	auto animationEntity3 = ECS::Instance()->createEntity("animatedModel3");
+	animationEntity3->addComponent<TransformComponent>();
+	animationEntity3->getComponent<TransformComponent>()->translate(3, 4, 2);
+	animationEntity3->getComponent<TransformComponent>()->translate(110.f, 100.f, 100.f);
+	animationEntity3->getComponent<TransformComponent>()->scale(0.005f);
+	animationEntity3->addComponent<ModelComponent>(&m_app->getResourceManager().getModel("AnimationTest/BaseMesh_Anim.fbx", shader));
+	animationEntity3->getComponent<ModelComponent>()->getModel()->setIsAnimated(true);
+	animationEntity3->addComponent<AnimationComponent>(&m_app->getResourceManager().getAnimationStack("AnimationTest/BaseMesh_Anim.fbx"));
+	animationEntity3->getComponent<AnimationComponent>()->currentAnimation = animationEntity3->getComponent<AnimationComponent>()->getAnimationStack()->getAnimation(0);*/
 #ifndef _DEBUG
-		//auto animationEntity4 = ECS::Instance()->createEntity("animatedModel4");
-		//animationEntity4->addComponent<TransformComponent>();
-		//animationEntity4->getComponent<TransformComponent>()->translate(-1,2,-3);
-		//animationEntity4->getComponent<TransformComponent>()->translate(110.f, 100.f, 100.f);
-		//animationEntity4->getComponent<TransformComponent>()->scale(0.005f);
-		//animationEntity4->addComponent<ModelComponent>(&m_app->getResourceManager().getModel("AnimationTest/DEBUG_BALLBOT.fbx", shader));
-		//animationEntity4->getComponent<ModelComponent>()->getModel()->setIsAnimated(true);
-		//animationEntity4->addComponent<AnimationComponent>(&m_app->getResourceManager().getAnimationStack("AnimationTest/DEBUG_BALLBOT.fbx"));
-		//animationEntity4->getComponent<AnimationComponent>()->currentAnimation = animationEntity4->getComponent<AnimationComponent>()->getAnimationStack()->getAnimation(0);
+	//auto animationEntity4 = ECS::Instance()->createEntity("animatedModel4");
```

Comment:
Remove unused code

Reply:
H3nx will remove or use this in his branch